src/vivarium_nih_us_cvd/components/healthcare_utilization.py

```diff
@@ -286,6 +374,26 @@ def determine_followups_ldlc(self, pop_visitors: pd.DataFrame) -> pd.Index:
 
         return needs_followup
 
+    def determine_followups_lifestyle(self, pop_visitors: pd.DataFrame) -> pd.Index:
```

Comment:
This is duplicated code. Isn't it identical to `treatment.enroll_in_lifestyle`? It may make more sense just to move enrollment of simulants in the lifestyle intervention into this component do avoid the duplication.